lib/Parser.js

```diff
@@ -105,13 +105,21 @@ class VEDirectParser extends EventEmitter {
     let converted = null
 
     if (!Array.isArray(this.line) || this.line.length !== 2) {
-      return this.warn('_parse() called on invalid line: ' + JSON.stringify({ line: this.line }))
+      if (this.options.ignoreChecksum !== true) {
+         return this.warn('_parse() called on invalid line: ' + JSON.stringify({ line: this.line }))
```

Comment:
Why is the `ignoreCheckSum` condition here? The line is invalid, not related to checksum, right?

Reply:
Regarding a test or test data: Is there an example I could have a look at on how to include such a test?

Reply:
Regarding ignoreCheckSum and the invalid line check: You are right, suppressing the warnings is actually not in line with the implicit meaning of "ignoreCheckSum".
If not suppressed, the plugin floods syslog. Maybe better to include a dedicated verbosity toggle for that? Or link this to the existing debug-toggle?

Reply:
test: https://github.com/SignalK/vedirect-serial-usb/blob/master/test/index.js

`warn` actually already calls debug. Where does the syslog spam come from? Do the emitted `warn`  events get logged?

Reply:
Yes, they end up in syslog. But I found the reason for that: I had set DEBUG=signalk-vedirect-parser and forgot about that.
I will revert the associated changes in the PR and make myself familiar with the tests.